Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -303,34 +320,39 @@ public override void StartDriver()
                         if (!(String.IsNullOrEmpty(SeleniumUserArguments) && String.IsNullOrWhiteSpace(SeleniumUserArguments)))
                             ieoptions.BrowserCommandLineArguments += "," + SeleniumUserArguments;
 
-                        if (Use64Bitbrowser == true)
+                        InternetExplorerDriverService IEService = null;
+
+                        if (Use64Bitbrowser)
                         {
                             string IEdriver64bitpath = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Locati‌​on) + @"\Drivers\IE64BitDriver");
+                            IEService = InternetExplorerDriverService.CreateDefaultService(IEdriver64bitpath);
+                        }
+                        else
+                        {
+                            IEService = InternetExplorerDriverService.CreateDefaultService();
+                        }
 
-                            if (Convert.ToInt32(HttpServerTimeOut) > 60)
-                                Driver = new InternetExplorerDriver(IEdriver64bitpath, ieoptions, TimeSpan.FromSeconds(Convert.ToInt32(HttpServerTimeOut)));
-                            else
-                                Driver = new InternetExplorerDriver(IEdriver64bitpath, ieoptions);
+                        IEService.HideCommandPromptWindow = HideConsoleWindow;
+                        if (Convert.ToInt32(HttpServerTimeOut) > 60)
```

Comment:
why only if bigger than 60? why not to check if != 60?
*this question/comment is for all HttpServerTimeOut IF's